src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/ParameterCapturingTests.cs

```diff
@@ -80,79 +86,75 @@ public async Task NonAspNetAppFailsOperation(Architecture targetArchitecture)
-        public async Task StopsProducingLogStatementsAfterOperationCompletes(Architecture targetArchitecture)
+        private async Task CapturesParametersCore(Architecture targetArchitecture, CapturedParameterFormat format)
         {
-            await RunTestCaseCore(TestAppScenarios.ParameterCapturing.SubScenarios.DoNotExpectLogStatement, targetArchitecture, async (appRunner, apiClient) =>
+            await RunTestCaseCore(TestAppScenarios.ParameterCapturing.SubScenarios.AspNetAppWithSampleMethod, targetArchitecture, async (appRunner, apiClient) =>
             {
                 int processId = await appRunner.ProcessIdTask;
 
                 CaptureParametersConfiguration config = GetValidConfiguration();
 
-                OperationResponse response = await apiClient.CaptureParametersAsync(processId, TimeSpan.FromSeconds(2), config);
+                OperationResponse response = await apiClient.CaptureParametersAsync(processId, TimeSpan.FromSeconds(2), config, format, FileProviderName);
                 Assert.Equal(HttpStatusCode.Accepted, response.StatusCode);
 
+                OperationStatusResponse operationStatus = await apiClient.WaitForOperationToStart(response.OperationUri);
+                Assert.Equal(OperationState.Running, operationStatus.OperationStatus.Status);
+
+                await appRunner.SendCommandAsync(TestAppScenarios.ParameterCapturing.Commands.CallMethod);
+
                 OperationStatusResponse operationResult = await apiClient.PollOperationToCompletion(response.OperationUri);
                 Assert.Equal(HttpStatusCode.Created, operationResult.StatusCode);
                 Assert.Equal(OperationState.Succeeded, operationResult.OperationStatus.Status);
 
-                await appRunner.SendCommandAsync(TestAppScenarios.ParameterCapturing.Commands.Validate);
-            });
-        }
-#else // NET7_0_OR_GREATER
-        [Theory(Skip = "The test will be replaced in a future commit to properly verify the new parameter capturing behavior.")]
-        [MemberData(nameof(ProfilerHelper.GetArchitecture), MemberType = typeof(ProfilerHelper))]
-        public async Task Net6AppFailsOperation(Architecture targetArchitecture)
-        {
-            await RunTestCaseCore(TestAppScenarios.ParameterCapturing.SubScenarios.AspNetApp, targetArchitecture, async (appRunner, apiClient) =>
-            {
-                int processId = await appRunner.ProcessIdTask;
-
-                CaptureParametersConfiguration config = GetValidConfiguration();
+                await appRunner.SendCommandAsync(TestAppScenarios.ParameterCapturing.Commands.Continue);
 
-                OperationResponse response = await apiClient.CaptureParametersAsync(processId, TimeSpan.FromSeconds(1), config);
-                Assert.Equal(HttpStatusCode.Accepted, response.StatusCode);
+                Assert.True(File.Exists(operationResult.OperationStatus.ResourceLocation));
```

Comment:
Please add an Assert.NotNull for this same property before asserting that the file exists.